proto/p4/v1/p4runtime.proto

```diff
@@ -394,8 +394,19 @@ message Replica {
 // each replica's egress input metadata will be set to the respective values
 // programmed in the multicast group entry.
 message MulticastGroupEntry {
+  // Set multicast_group_id to MULTICAST_GROUP_ID_WILDCARD for wildcard reads
   uint32 multicast_group_id = 1;
   repeated Replica replicas = 2;
+
+  // We define a special reserved value for the multicast_group_id field. This
+  // value can be used to do a wildcard Read on all configured multicast
+  // groups. This value cannot be used in Write RPCs.
+  enum MulticastGroupIdReserved {
+    MULTICAST_GROUP_ID_INVALID = 0;
+    MULTICAST_GROUP_ID_FIRST = 1;
```

Comment:
What do we need `MULTICAST_GROUP_ID_FIRST` for?  In particular it doesn't seem to be reserved, unlike the name suggests.

Reply:
I added it for symmetry with clone sessions, for which it is required since the enum must start with a value of 0.

Agreed on the naming. I replaced "Reserved" with "Values" in both cases.